aptos-move/aptos-vm/src/move_vm_ext/vm.rs

```diff
@@ -32,6 +34,7 @@ impl MoveVmExt {
         treat_friend_as_private: bool,
         allow_binary_format_v6: bool,
         chain_id: u8,
```

Comment:
With `features` present, we can additionally get rid of parameters `treat_friend_as_private` and `allow_binary_format_v6`? If we want.